ecosystem/node-checker/README.md

```diff
@@ -4,42 +4,135 @@ The Aptos Node Health Checker (NHC) is the reference implementation of a node he
 - **Operator Support**: As node operators, you will want to know whether your node is running correctly. This service can help you figure that out. While we host our own instances of this service, we encourage node operators to run their own instances. You may choose to either run a publicly available NHC or run it as a sidecar, where it only works against your own node.
 - **Continuous Evaluation**: As part of the AITs, Aptos Labs needs a tool to help confirm that participants are running their nodes in a way that meets our criteria. We run this tool continuously throughout each AIT to help us evaluate this.
 
-In this README we describe how to run NHC in each of the above configuration.
+In this README we describe how to run NHC for the **Operator Support** use case. NHC can reasonably be run both as an external tool as well as a sidecar process for this use case. Both are described below. For more information on how NHC works, please see [How NHC works](#how-nhc-works) below.
 
-## Running NHC
-For now, the best way to run NHC is to build and run it locally.
+## tl;dr
+While we highly recommend you read this whole README, you can get NHC working in a basic form by doing the following. This baseline configuration is for a devnet fullnode.
 
-First, check out the repo and navigate here.
+Get a baseline configuration:
+```
+cd /tmp/nhc && wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configurations/devnet_fullnode.yaml
+```
+
+Run NHC:
```

Comment:
Then, start the NHC service by providing the above-downloaded `devnet_fullnode.yaml` configuration file.